client/client.go

```diff
@@ -41,7 +41,7 @@ func Build(URL, version string) (*resty.Client, error) {
 		}).
 		SetHeader("content-type", "application/json").
 		SetHeader("accept", "*/*").
-		SetHeader("user-agent", "jfrog/terraform-provider-artifactory:"+version).
+		SetHeader("user-agent", "jfrog/"+productId).
```

Comment:
That's exactly what I had in mind. Of course, now you need to version bump things :)